grammars/asciidoc.cson

```diff
@@ -30,7 +30,7 @@
   #
   #   === Level 2 Section
   {
-    'begin': '^={1,6}\\p{Blank}+'
+    'begin': '^(={1,6}|[#]{1,6})([ \\t]+)'
```

Comment:
The second expression breaks the tests.
I've reverted to the first proposition.